Rakefile

```diff
@@ -31,4 +31,4 @@ end
 require 'rubocop/rake_task'
 RuboCop::RakeTask.new
 
-task default: [:rubocop, :spec]
+task default: :spec
```

Comment:
@dblock are you okay with this? I think this might be suprizing for people who use Rubocop regularly.

Reply:
I would leave the default as is and run both. This is just because it has been this way for a long time and we like having rubocop in the mongoid org.